redaxo/src/addons/media_manager/lib/media_manager.php

```diff
@@ -75,19 +89,23 @@ public function getMedia()
         return $this->media;
     }
 
+    /**
+     * @param string $type
+     */
     protected function applyEffects($type)
     {
         $this->type = $type;
 
         if (!$this->isCached()) {
             $set = $this->effectsFromType($type);
+            /** @var list<array{effect: string, params: array<string, mixed>}> $set */
             $set = rex_extension::registerPoint(new rex_extension_point('MEDIA_MANAGER_FILTERSET', $set, ['rex_media_type' => $type]));
 
             if (0 == count($set)) {
                 $this->useCache = false;
                 $this->notFound = !$this->media->exists();
 
-                return $this->media;
+                return;
```

Comment:
Die Methode ist eigentlich void, und nur in diesem einen Fall hier wurde das Objekt geliefert.
Das macht für mich so keinen Sinn, und den Wert nehmen wir auch nirgends entgegen.